src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1028,6 +1028,12 @@ exprt java_bytecode_convert_methodt::get_or_create_clinit_wrapper(
     code_function_callt call_real_init;
     call_real_init.function()=findsymit->second.symbol_expr();
     init_body.move_to_operands(call_real_init);
+
+    if(lazy_methods)
+      // lazy methods enabled and the clinit
```

Comment:
This comment doesn't say what's happening here. This is requesting *this* clinit be loaded; is the call to a parent clinit missing, in which case that's a separate bug?

Reply:
yep sorry, there is is no additional bug, as   get_or_create_clinit_wrapper, will recurse on the parents of the class in question.